packages/@lwc/engine-core/src/framework/vnodes.ts

```diff
@@ -74,6 +75,7 @@ export interface VNodeData {
     readonly context?: Readonly<Record<string, Readonly<Record<string, any>>>>;
     readonly on?: Readonly<Record<string, (event: Event) => any>>;
     readonly svg?: boolean;
+    readonly isStatic?: boolean;
```

Comment:
Vue uses [a patch flag](https://github.com/vuejs/core/blob/245230e135152900189f13a4281302de45fdcfaa/packages/shared/src/patchFlags.ts#L114) to indicate that an element is hoisted/static. Maybe we should start using a system like this? Presumably it's more efficient than attaching members to an object.